Packs/fireeye/Integrations/integration-fireeye.yml

```
@@ -381,7 +382,7 @@ script:
     sendRequest(commandDictionary.logout.url, commandDictionary.logout.method, token);
 
 
-    if (typeof result === 'object' && !result['submission_Key']) {
+    if (typeof result === 'object' && !result['submission_Key'] && command != 'fe-submit-status') {
```

the purpose here is to update the submisison key upon a submission, correct? if so, isn;t it better to:
```suggestion
    if (typeof result === 'object' && !result['submission_Key'] && (command == 'fe-submit' || command == 'fe-submit-url')) {
```

---

The purpose here is to update the submission key wherever it missing. The integration worked just fine for all commands before the change I did for the fe-submit-status which broke this behavior for the command (since the returned result is a list instead of dict). Since I only changed (and broke) the `fe-submit-status` command this fix is only relevant for this command I am not sure on what other cases this statement is needed (I can imagine but I don't think it is smart to assume here)